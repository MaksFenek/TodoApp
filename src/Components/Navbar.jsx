// ==== React imports ====
import React from 'react';

// ==== Style imports ====
import '../Styles/navbar.scss';

export default function navbar() {
  return (
    <nav className='nav'>
      <div className='nav-logo'>
        <h1>ToDo App</h1>
      </div>
    </nav>
  );
}
