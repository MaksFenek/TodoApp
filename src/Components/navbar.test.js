import React from 'react';
import Navbar from './Navbar';
import renderer from 'react-test-renderer';

describe('Navbar', () => {
  it('should render corrently', () => {
    const tree = renderer.create(<Navbar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
