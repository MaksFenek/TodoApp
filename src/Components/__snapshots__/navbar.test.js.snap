// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render corrently 1`] = `
<nav
  className="nav"
>
  <div
    className="nav-logo"
  >
    <h1>
      ToDo App
    </h1>
  </div>
</nav>
`;
